chat-plugins/chat-monitor.js

```diff
@@ -6,7 +6,7 @@ const MONITOR_FILE = 'config/chat-plugins/chat-monitor.tsv';
 const WRITE_THROTTLE_TIME = 5 * 60 * 1000;
 
 /** @type {{[k: string]: string[]}} */
-let filterKeys = Chat.filterKeys = Object.assign(Chat.filterKeys, {publicwarn: ['PUBLIC', 'WARN', 'Filtered in public'], warn: ['EVERYWHERE', 'WARN', 'Filtered'], autolock: ['EVERYWHERE', 'AUTOLOCK', 'Autolock'], namefilter: ['NAMES', 'WARN', 'Filtered in usernames'], wordfilter: ['EVERYWHERE', 'FILTERTO', 'Filtered to a different phrase']});
+let filterKeys = Chat.filterKeys = Object.assign(Chat.filterKeys, {publicwarn: ['PUBLIC', 'WARN', 'Filtered in public'], warn: ['EVERYWHERE', 'WARN', 'Filtered'], autolock: ['EVERYWHERE', 'AUTOLOCK', 'Autolock'], namefilter: ['NAMES', 'WARN', 'Filtered in usernames'], wordfilter: ['EVERYWHERE', 'FILTERTO', 'Filtered to a different phrase'], shorteners: ['EVERYWHERE', 'SHORTENER', 'Url Shorteners']});
```

Comment:
I'm trying to enforce a line length limit right now.

```suggestion
let filterKeys = Chat.filterKeys = Object.assign(Chat.filterKeys, {
	publicwarn: ['PUBLIC', 'WARN', 'Filtered in public'],
	warn: ['EVERYWHERE', 'WARN', 'Filtered'],
	autolock: ['EVERYWHERE', 'AUTOLOCK', 'Autolock'],
	namefilter: ['NAMES', 'WARN', 'Filtered in usernames'],
	wordfilter: ['EVERYWHERE', 'FILTERTO', 'Filtered to a different phrase'],
	shorteners: ['EVERYWHERE', 'SHORTENER', 'Url Shorteners'],
});
```

Reply:
done